$OpenBSD: patch-configure,v 1.6 2014/12/21 21:22:52 sthen Exp $

Use GNU readline as our base readline lacks add_history_time().
Fix failing test for libssl, LibreSSL doesn't provide RAND_egd.

--- configure.orig	Mon Oct 13 11:41:55 2014
+++ configure	Sun Dec 21 22:01:12 2014
@@ -48087,7 +48087,7 @@ if ${lftp_cv_precompiled_readline+:} false; then :
 else
 
    old_LIBS="$LIBS"
-   LIBS="-lreadline $READLINE_SUPPLIB $LIBS"
+   LIBS="-lereadline $READLINE_SUPPLIB $LIBS"
    cat confdefs.h - <<_ACEOF >conftest.$ac_ext
 /* end confdefs.h.  */
 extern int (*rl_getc_function)();
@@ -48127,7 +48127,7 @@ if test $lftp_cv_precompiled_readline = yes; then
    READLINE_DIR=''
    { $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
 $as_echo "yes" >&6; }
-   READLINE='-lreadline'
+   READLINE='-lereadline'
    READLINE_DEPEND=''
    COMPILE_READLINE=no
 else
@@ -48420,7 +48420,7 @@ else
 int
 main ()
 {
-static SSL_CTX *ctx; SSL_new(ctx); RAND_status(); RAND_egd("file")
+static SSL_CTX *ctx; SSL_new(ctx); RAND_status();
   ;
   return 0;
 }
